_pages/about-us/teams/products-and-platforms.md

```diff
@@ -51,10 +53,10 @@ If you're a P&P project lead, expect to:
 
 * Report weekly on your project's status, blockers, and financial status via the weekly meeting agenda.
 
-* Be involved in the staffing process
+* Be involved in the staffing process.
```

Comment:
This was one action item I felt could use more clarification (or perhaps be broken into separate items). What exactly does it mean to be involved in the staffing process?